qiskit/schemas/qobj_schema.json

```diff
@@ -999,16 +999,141 @@
+                                            },
+                                            "type": "array"
+                                        }
+                                    },
+                                    "required": [
+                                        "h_str",
+                                        "vars",
+                                        "dim_osc",
+                                        "dim_qub"
+                                    ]
+                                },
+                                "dt": {
+                                    "minimum": 0,
+                                    "type": "number"
+                                },
+                                "noise_model": {
+                                    "properties": {
+                                        "qubit": {
+                                            "description": "Noise models for qubit.",
+                                            "items": {
+                                                "type": "object",
+                                                "patternProperties": {
+                                                    "^([A-Za-z])+$": {
+                                                        "type": "number"
+                                                    }
+                                                },
+                                                "additionalProperties": false
+                                            },
+                                            "type": "array",
+                                            "additionalItems": false
+                                        },
+                                        "oscillator": {
+                                            "description": "Noise models for oscillator.",
+                                            "items": {
+                                                "items": [
+                                                    {
+                                                        "type": "number"
+                                                    },
+                                                    {
+                                                        "type": "number"
+                                                    }
+                                                ],
+                                                "type": "array",
+                                                "additionalItems": false
+                                            },
+                                            "type": "array"
+                                        }
+                                    },
+                                    "required": [
+                                        "qubit",
+                                        "oscillator"
+                                    ]
+                                },
+                                "ode_options": {
```

Comment:
These should also be removed. Aer can always provide its own jsonschema with specific `ode_options` if required.